fnet/network.go

```diff
@@ -25,6 +25,19 @@ import (
 	"istio.io/fortio/version"
 )
 
+const (
+	// DefaultGRPCPort is the Fortio gRPC server default port number.
+	DefaultGRPCPort = "8079"
+	// DefaultHTTPPort is the Fortio http server default port number.
```

Comment:
that comment isnt correct 
call 80 and 443 StandardHTTP(S)Port and the comment is it is the standard

unlike 8079 which is the default port the grpc server listens to (the default http is 8080)